helpers/errorhandlers.go

```diff
@@ -20,11 +20,11 @@ import (
 	ctrl "sigs.k8s.io/controller-runtime"
 )
 
-func (l IntegrationLogger) HandleLoaderError(err error, resource, from string) (ctrl.Result, error) {
+func HandleLoaderError(logger IntegrationLogger, err error, resource, from string) (ctrl.Result, error) {
```

Comment:
I cannot comment there, but copyright on line 2 should include owner:  `Copyright 2023 Red Hat Inc.`
There is another PR which is fixing this